tests/network-tests/src/flows/clis/initStorageBucket.ts

```diff
@@ -34,7 +34,7 @@ export default async function initStorageBucket({ api }: FlowProps): Promise<voi
     transactorKey,
   ])
   await cli.run('leader:update-bag', ['--add', bucketId, '--bagId', 'static:council'])
-  await cli.run('leader:update-dynamic-bag-policy', ['--bagType', 'Channel', '--number', '1'])
+  await cli.run('leader:update-dynamic-bag-policy', ['--bagType', 'Channel', '--number', '5'])
```

Comment:
This change should no longer be needed after https://github.com/Joystream/joystream/pull/4007